requirements.txt

```diff
@@ -1,4 +1,4 @@
-qiskit-terra>=0.18.0
+qiskit-terra>=0.18.0,<0.20.0
```

Comment:
Same as https://github.com/Qiskit/qiskit-ibm-provider/pull/312#discussion_r841499728. I don't see why this is needed from looking at this file alone.